runtests.py

```diff
@@ -949,6 +953,14 @@ def skip_c(tags):
                     return True
     return False
 
+def skip_limited(tags):
+    if sys.version_info[0] < 3:
+        return True
+    if sys.implementation.name == 'cpython':
+        return False
+    # on all non-cpython, skip limited-api tests
+    if 'limited-api' in tags['tag']:
+        return True
 
```

Comment:
Let's leave a note that we didn't just forget to handle other cases.
```suggestion
        return True
    return False
```